exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,13 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (name === "Daniel" && danielsRole == "mentor") {
```

Comment:
Nice!
The true part of the IF statement **returns** a value, and the ELSE calls **console.log**, what is the requirement and which one should  both parts of the IF be doing?

Reply:
The requireement is for both parts of the IF to be true. If either one is false, it will fall to the "else" part of the statement.

Reply:
Ok. I just noted that the if the _name === "Daniel" and danielsRole === "mentor"_, it `returns `something, otherwise it `console.logs `something.
Don't you think it would be better if both  parts of the if ( if and else ) does the same thing, either `return `or `console.log`?